V1_EngagingWithUs.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: engaging-with-us
+is: 'published'
+---
+
+## Engaging with us
+
+We are continually adding value and want to include you as early as possible in our development process so we can include 
+your input and feedback. Some features are stable (in that they will not change), others are in progress of being built which we call **beta** 
+and others are things we want to try and discover what would work best, we call **experimental**. Additionally, when we are about to 
+introduce something new and want to get feedback on the concepts and understanding before we expose API’s, so that your 
+feedback can inform and guide the future API, we will call these **future**. Also, **future** may, (but not always), include the ability for you to 
+play with an API, through one of our tools (like the GraphQL Explorer), so you can have some experience with it and give us 
+feedback to help inform and improve things.
+
```

Comment:
Add here a reference to how to change the level of APIs exposed in API Explorer.  We just delivered this functionality which should also be referenced in the "What's New" doc.  @nivekselig can provide details

Reply:
ok, all set